extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/runtime/OidcClientRecorder.java

```diff
@@ -147,6 +149,31 @@ public OidcClient apply(String tokenRequestUri) {
         });
     }
 
+    private static void setHttpsClientOptions(URI authServerUri, WebClientOptions options) {
```

Comment:
@sd-f can you please rename it to `setWebClientOptions` ? It is not only https properties that are set there, right now `trustAll` is dealt in `OidcCommonUtils`...

Reply:
@sberyozkin done :)